src/routers/uploadProfile.ts

```diff
@@ -1,5 +1,6 @@
 import express from 'express'
 import { uploadImages } from '../controllers/uploader'
+import tokenVerify from '../middlewares/tokenVerify'
```

Comment:
You imported but forgot to use

Reply:
if i use this, then client side say unauthorized
If i use axios interceptor then aws doens't accept the request

Reply:
But if you're not using middleware, then how are you receiving the user id ?

Reply:
Yes i understood, but there is side effect :) 
BTW , i used now on upcoming commit